tests/Unit/DataStructures/Test_Variables.cpp

```diff
@@ -229,30 +229,51 @@ void test_variables_construction_and_access() noexcept {
       == "Variables(vector,scalar,scalar2)");
 }
 
-SPECTRE_TEST_CASE("Unit.DataStructures.Variables.Move",
-                  "[DataStructures][Unit]") {
-  Variables<tmpl::list<VariablesTestTags_detail::vector>> x(1, -2.0),
-      z(2, -3.0);
-  CHECK(&get<VariablesTestTags_detail::vector>(z)[0][0] == z.data());
-  Variables<tmpl::list<VariablesTestTags_detail::vector>> y = std::move(x);
-  x = std::move(z);
-  CHECK(
-      (x == Variables<tmpl::list<VariablesTestTags_detail::vector>>{2, -3.0}));
-  CHECK(&get<VariablesTestTags_detail::vector>(x)[0][0] == x.data());
+template <typename VectorType>
+void test_variables_move() noexcept {
+  MAKE_GENERATOR(gen);
+  UniformCustomDistribution<
+      typename tt::get_fundamental_type_t<typename VectorType::value_type>>
+      dist{-100, 100};
+  UniformCustomDistribution<size_t> sdist{2, 20};
```

Comment:
Let's do 5 -> 20 :) We've had 2 points let us sneak problems into the code in the past